package.json

```diff
@@ -203,7 +202,7 @@
     "eslint-plugin-jest": "^27.2.3",
     "husky": "^8.0.3",
     "jest": "^29.6.1",
-    "jest-preset-stylelint": "^6.1.0",
+    "jest-preset-stylelint": "github:stylelint/jest-preset-stylelint#getOsEol-not-needed",
```

Comment:
- [ ] Wait for a new version of `jest-preset-stylelint`

Reply:
Updated to https://github.com/stylelint/jest-preset-stylelint/releases/tag/6.1.1 via 02ed7173b